### ISSUE TITLE
Use Alias rather than Redirect for /Consortium/Process

### ISSUE THREAD

**Issue by fantasai** [w3c/process#729]
Currently /Consortium/Process/ performs a client-side redirect. This makes it very hard to link to the Process, since you're always on a dated snapshot. We should be _encouraging_ links to the undated Process, not discouraging them in favor of dated snapshots. :/

**Comment by frivoal** [comment]
(marked as deferred not to say we should wait before we do anything about this, but in the sense that this is independent of the P2023 cycle, and therefore does not need to block it).

**Comment by plehegar** [comment]
My take would be to do sa we do for /TR documents. Ie, no redirect.

**Comment by koalie** [comment]
> My take would be to do as we do for /TR documents. Ie, no redirect.

That would work for W3C Comm.

**Comment by plehegar** [comment]
So, /Consortium/* is getting retired following the new website redesign.

So, here is what we could do:

1. Redirect /Consortium/Process -> /Process

2. Redirect on /Process like we do for /TR

**Comment by fantasai** [comment]
What is the rest of the related URL structures? I don't think Process should be top-level, it should be on the same level as the Bylaws, Patent Policy, Articles of Incorporation, etc. They should all be grouped together...

**Comment by fantasai** [comment]
Also could someone explain why we're deprecating URLs as part of the redesign? Is it because we have a better URL scheme in mind (what is it?) or because there's some technical limitation that requires all the new pages to be on new URLs?

**Comment by glorialilly727** [comment]
Someone keeps changing passwords and glorialilly836@gmail.com and glorialilly727@gmail.com was old Gmail account and data breaches internet Wi-Fi and website my inheritance Information on my accounts and Gmails accounts etc itenty theft and all life lock Norton Mcfee protection accounts stolen from my phone I need help with information FBI government law enforcement about issue wifi internet Xfinity Comcast hotspot glorialilly727@gmail.com military family and all information I need find please contact proper authoritys if login on after me

**Comment by plehegar** [comment]
> Also could someone explain why we're deprecating URLs as part of the redesign? Is it because we have a better URL scheme in mind (what is it?) or because there's some technical limitation that requires all the new pages to be on new URLs?

That seems outside the scope of the W3C Process repository issues.

**Comment by koalie** [comment]
I have consulted with @deniak who confirms no redirect to the dated version of the process but using a proxy instead.
Before doing so, I need to close the loop with @vivienlacourba on whether we keep the existing URL or not and if not which to use. I expect this can be done within the next few weeks (to give everyone time to come back from holiday and be up to speed).

Regarding the question raised by @fantasai pertaining to the URL strategy, please refer to the [recommendation](https://w3c.studio24.net/docs/url-strategy) from the agency we worked with for the redesign of the website.

**Comment by plehegar** [comment]
@frivoal @fantasai , any reason not to do this ?

**Comment by frivoal** [comment]
I had missed this earlier. I like most of the proposal, with one exception:


>     * `/policies/process/` -> will proxy (and not redirect) to the latest published dated version (replaces [/Consortium/Process/](https://www.w3.org/Consortium/Process/))
> 
>     * `/policies/process/YYYYMMDD/` (replaces [/2023/Process-20231103/](https://www.w3.org/2023/Process-20231103/))
> 
>     * `/policies/process/drafts/` -> will proxy (and not redirect) to the editors draft on github.io https://w3c.github.io/w3process/ (replaces: [/Consortium/Process/Drafts/](https://www.w3.org/Consortium/Process/Drafts/))

All the above seems good.

>     * `/policies/process/drafts/YYYYMMDD/` (replaces [/Consortium/Process/Drafts/snapshots/2023-08](https://www.w3.org/Consortium/Process/Drafts/snapshots/2023-08))

I'm less sure about this one. Currently, anything under `Draft/` (including `Draft/snapshots/*`, but also `Draft/issues-*`, as well as built (historical) branches such as `Draft/everblue`, `Draft/tooling`, `Draft/registries`…) is not managed by a formal process, but simply proxying the content of this repo. Moving the whole thing from `Draft/` to `drafts/` seems fine, but I'd leave everything under that as an unmanaged space proxying the whole of what's under https://w3c.github.io/w3process/.

If you do want to put a bit of order into what's in that space, I think we could talk about that too, but as there's a lot of things in that space, I'd suggest a separate issue to discuss it.

**Comment by fantasai** [comment]
@vivienlacourba Where will the Bylaws live? I think Process probably belongs in a parallel location...

**Comment by plehegar** [comment]
Looking above, we agreed on:

>     * `/policies/process/` -> will proxy (and not redirect) to the latest published dated version (replaces [/Consortium/Process/](https://www.w3.org/Consortium/Process/))
> 
>     * `/policies/process/YYYYMMDD/` (replaces [/2023/Process-20231103/](https://www.w3.org/2023/Process-20231103/))
> 
>     * `/policies/process/drafts/` -> will proxy (and not redirect) to the editors draft on github.io https://w3c.github.io/w3process/ (replaces: [/Consortium/Process/Drafts/](https://www.w3.org/Consortium/Process/Drafts/))


What happens below /policies/process/drafts/* will continue to be managed by that repo.

Unless we hear otherwise, we'll close this issue with this conclusion.

**Comment by frivoal** [comment]
> Where will the Bylaws live? I think Process probably belongs in a parallel location...

Currently they're at https://www.w3.org/2022/10/w3c-bylaws-20221019. It might be reasonable to move them to that `/policies/*` space too indeed.

I note that the CoC already lives there (https://www.w3.org/policies/code-of-conduct/), as well as the antitrust policy (https://www.w3.org/policies/antitrust/) or the privacy policy (https://www.w3.org/policies/privacy/), which seems appropriate, but that the Patent Policy does not: It's at https://www.w3.org/Consortium/Patent-Policy/, and like the current Process (and unlike what we're proposing to do here), it redirects (not proxies) to the latest dated version.

I feel less strongly about the bylaws, but I'd suggest doing the same for the Patent Policy as we're proposing doing for the Process.

**Comment by plehegar** [comment]
Waiting on #871 to happen, then @vivienlacourba and @deniak will do the redirects for Process and PP, then update pubrules to accept old and new dated 20231106 process version and old/new patent policy links, and then do pull requests for bikeshed/respec.

**Comment by koalie** [comment]
Hi all,
@vivienlacourba and I met today (after he met yesterday with @plehegar) to decide on the following implementation details:

1. "this version" URI + "latest" URI to be edited in place; entry to be added in the "changes" section with date of change and explanation
2. no need to perform the same change of URIs for previous versions
3. this applies to both the Process Document and the Patent Policy

**Comment by frivoal** [comment]
@koalie I think you mean this needs to happen not only in the Editor's Draft, but also in the published versions, is that right? The team will need to handle the already published version, but I'm happy to handle the Editor's Drafts. Just let me know:
* when we're ready to go live with this
* if there's a particular phrasing you'd like to see in the changelog explaining why we're doing this

**Comment by koalie** [comment]
> @koalie I think you mean this needs to happen not only in the Editor's Draft, but also in the published versions, is that right? 

Yes the editors draft should be updated accordingly to reflect this change.

> The team will need to handle the already published version, but I'm happy to handle the Editor's Drafts. Just let me know:
> 
>     * when we're ready to go live with this

Will do!

>     * if there's a particular phrasing you'd like to see in the changelog explaining why we're doing this

@vivienlacourba suggested wording which I like: " this document URI was changed from /Consortium/Process/ to /policies/process/ for better integration in the W3C website architecture."

**Comment by deniak** [comment]
The process document has now been moved under /policies:

- https://www.w3.org/policies/process/
- https://www.w3.org/policies/process/20231103/
- https://www.w3.org/policies/process/drafts/

**Comment by vivienlacourba** [comment]
> The process document has now been moved under /policies:
> 
>     * https://www.w3.org/policies/process/
> 
>     * https://www.w3.org/policies/process/20231103/
> 
>     * https://www.w3.org/policies/process/drafts/

Thank you @deniak !

@plh @frivoal you can now take care of updating the editors draft accordingly with the new URIs of the Process doc and patent policy.

**Comment by frivoal** [comment]
> You can now take care of updating the editors draft accordingly with the new URIs of the Process doc and patent policy.

Done.

**Comment by frivoal** [comment]
@vivienlacourba Thanks, nice catch. Should be fixed now.
